README.md

```diff
@@ -1 +1,9 @@
-# ehdsi
\ No newline at end of file
+# ehdsi
+
+## Test server setup
+Make sure that the user that is running the docker containers has access to the docker socket.
+
+```bash
+sudo usermod -aG docker $USER
+```
+
```

Comment:
This needs some more context.  You made a "Documentation" directory at some point for notes like this, should it be there instead?

I would like the first text that goes into the root README to at least provide a description of the content of the repo.